ResearchKit/ActiveTasks/ORKTowerOfHanoiTowerView.m

```diff
@@ -66,7 +66,8 @@ - (instancetype)initWithFrame:(CGRect)frame maximumNumberOfDisks:(NSUInteger)max
 
 - (void)updateConstraints {
     if (_currentConstraints != nil) {
-        [NSLayoutConstraint deactivateConstraints:_currentConstraints];
+        [self removeConstraints:_currentConstraints];
```

Comment:
Use `removeConstraints:` to be consistency with other UIView subclass like `ORKSurveyAnswerCellForPicker`. 
And happens to fixed this crash.